input/fsh/DEF_CodeSystems.fsh

```diff
@@ -21,3 +21,24 @@ Description: "Codes requested from SNOMED by CodexRT, to be issued at a later da
 * #1259981006 "Radiotherapy course change due to change to treatment intent (finding)" "Radiotherapy course change due to change to treatment intent (finding)" //#USCRS-34603
 * #1259231006 "Radiotherapy course change due to changes to target (finding)" "Radiotherapy course change due to changes to target (finding)" // USCRS-34604
 * #1259232004 "Radiotherapy course change due to change to organ at risk (finding)" "Radiotherapy course change due to change to organ at risk (finding)." // USCRS-34605
+
+CodeSystem: RadiotherapyRespiratoryMotionManagementCS
+Id: codexrt-radiotherapy-respiratory-motion-management-cs
+Title: "Respiratory Motion Management"
+Description: "Method applied to manage respiratory motion."
+* ^caseSensitive = true
+* ^experimental = false
+* #free-breathing "Radiotherapy with free-breathing" "Radiotherapy with free-breathing"
+* #breath-hold "Radiotherapy with breath-hold" "Radiotherapy with breath-hold"
+* #none "No respiratory motion management" "Radiotherapy without respiratory motion management"
+
+CodeSystem: RadiotherapyFreeBreathingMotionManagementTechniqueCS
+Id: codexrt-radiotherapy-free-breathing-motion-mgmt-technique-cs
```

Comment:
Since we will be submitting SNOMEDCT concept requests for these, recommend moving these to a SnomedRequestCS and using the USCRS # for the code for now and including the concept type in the display name.

For example #free-breathing --> USCRS-XXX Radiotherapy with free breathing (regime/therapy)